### PR TITLE
145 reducer refactor

### PR DESCRIPTION
**Pull request by afletch24**
Reviewed reducer files and changed format when necessary.

### CODE REVIEW

File: src/reducers/meeting_options_reducer.js [hunk 1]
```diff
@@ -4,8 +4,7 @@ const INITIAL_STATE = [];
 export default (state = INITIAL_STATE, action) => {
   switch(action.type){
     case FETCH_MEETING_OPTIONS:
-      // return action.payload.data;
-      return [...state, ...action.payload.data];
+      return action.payload.data;
```
Comment (mkirlin) [inline src/reducers/meeting_options_reducer.js]: Should we do the same `...state` thing here that we're doing elsewhere? Is there some reason not to do so?

File: src/common/css/navbar.css [hunk 2]
```diff
@@ -5,6 +5,7 @@
 .navbar {
   padding-top: 10px;
   padding-bottom: 10px;
+  clear: both
```
Comment (vhoof) [inline src/common/css/navbar.css]: I think you want a semicolon.

File: src/reducers/feeds_reducer.js [hunk 3]
```diff
@@ -9,3 +9,5 @@ export default (state = INITIAL_STATE, action) => {
       return state;
   }
 };
+
+
```
Comment (vhoof) [inline src/reducers/feeds_reducer.js]: I think we can remove one of these newlines.

File: src/actions/event_actions.js [hunk 4]
```diff
@@ -12,6 +12,7 @@ import {
   UPDATE_EVENT_FAIL
 } from "./types";
 
+
```
Comment (vhoof) [inline src/actions/event_actions.js]: I think you can remove this extra newline.

File: src/AdminPage/components/EmailModal.js [hunk 5]
```diff
@@ -21,17 +21,20 @@ class EmailModal extends Component {
   handleClose() {
     this.setState({ open: false });
   }
-
+
```
Comment (vhoof) [inline src/AdminPage/components/EmailModal.js]: I think we want to try to avoid tabbed but otherwise empty lines, but it's not a big deal.

**Pull request merged**